Content.Shared/Tiles/FloorTileSystem.cs

```diff
@@ -37,6 +37,7 @@ public sealed class FloorTileSystem : EntitySystem
     [Dependency] private readonly TileSystem _tile = default!;
     [Dependency] private readonly SharedPhysicsSystem _physics = default!;
     [Dependency] private readonly SharedMapSystem _map = default!;
+    [Dependency] private readonly TileStackSystem _tileStack = default!;
```

Comment:
Tight coupling. TILE stacks, Floor TILE System.

See Door bolts (currently) and my partialization of Airlock and Firelock (PR) as examples of partialization. If a system is an orphan without its parent system and is just there to extend and customize an underlying system, it should be a partial.